clarity/src/vm/docs/contracts.rs

```diff
@@ -1,13 +1,13 @@
-use chainstate::stacks::boot::STACKS_BOOT_CODE_MAINNET;
+//use chainstate::stacks::boot::STACKS_BOOT_CODE_MAINNET;
```

Comment:
:eyes:

Reply:
This is dropped now -- docs tests and generation were reenabled in 84b4369485e13bc1127bb2eb65fbd053343917d2